editor/scene_tree_dock.cpp

```diff
@@ -4132,10 +4301,26 @@ SceneTreeDock::SceneTreeDock(Node *p_scene_root, EditorSelection *p_editor_selec
 	delete_tracks_checkbox->set_pressed(true);
 	vb->add_child(delete_tracks_checkbox);
 
+	editable_instance_enable_or_disable_dialog = memnew(ConfirmationDialog);
+	editable_instance_enable_or_disable_dialog->set_text(TTR("Selection contains some nodes that have \"Editable Children\" enabled and some that do not."));
+	editable_instance_enable_or_disable_dialog->set_ok_button_text(TTR("Enable for all"));
```

Comment:
Use Title Case for buttons:

```suggestion
	editable_instance_enable_or_disable_dialog->set_ok_button_text(TTR("Enable for All"));
```